snap/snapcraft.yaml

```diff
@@ -385,14 +741,159 @@ slots:
     name: org.freedesktop.ColorHelper
     bus: session
 
+lint:
+  ignore:
+    - classic
+
 parts:
+  snapbuildtools:
+    source: https://github.com/sergio-costas/snap-build-tools.git
+    source-branch: manage-dangling-symlinks
```

Comment:
If we want to keep this, it probably needs to move from your personal namespace.

Possibly putting the scripts in this module if it's not shared by many snaps.

Reply:
Put here by now, since I'm using a specific branch to manage the broken symlinks to icons.